packages/pageheader/components/PageHeader.tsx

```diff
@@ -14,30 +14,53 @@ import PageHeaderTabs from "../components/PageHeaderTabs";
 import { fillHeight } from "../style";
 
 export interface PageHeaderProps {
+  /**
+   * Breadcrumbs to display on the left side of the PageHeader.
+   */
   breadcrumbElements: React.ReactNodeArray;
+  /**
+   * Actions to display on the right side of the PageHeader.
+   */
   actions?: Array<React.ReactElement<any>>;
   children?: React.ReactNode;
+  /**
+   * Contains the header to a specified width.
+   */
+  maxWidth?: React.CSSProperties["maxWidth"];
```

Comment:
so... just because of weird content box / border box stuff, I'd actually suggest just doing this via className

Reply:
FWIW I did test it out with npm link 
<img width="1540" alt="Screen Shot 2022-11-07 at 9 34 57 AM" src="https://user-images.githubusercontent.com/34781875/200350497-8892c454-416c-4f98-a23f-9c66de7599b7.png">

Reply:
I think the FullscreenView is lacking border-box but the PageHeader does have it set around one of the outer divs.

Reply:
Is a classname still preferred here? 
It seems it would have to be set on the outermost div instead of this one. The outer cannot have the maxWdith set on it or the bottom border will not span the entire page. 

Then we'll likely need to pass something like this in the UI:
```
  const pageHeaderContainer = css`
    > div {
      max-width: "600px";
      margin: 0 auto;
    }
  `;
```

Reply:
I'm going to update this to a className along with some other changes.